client/grpc/tmservice/service_test.go

```diff
@@ -85,6 +85,21 @@ func (s IntegrationTestSuite) TestQueryLatestBlock() {
 	s.Require().NoError(val.ClientCtx.Codec.UnmarshalJSON(restRes, &blockInfoRes))
 }
 
+func (s IntegrationTestSuite) TestQueryBlockByHash() {
+	val := s.network.Validators[0]
+	node, _ := val.ClientCtx.GetNode()
+	blk, _ := node.Block(context.Background(), nil)
+	blkhash := blk.BlockID.Hash
+	res, err := s.queryClient.GetBlockByHash(context.Background(), &tmservice.GetBlockByHashRequest{Hash: blkhash})
+	s.Require().NoError(err)
+	_ = res
```

Comment:
If you don't want to use `res`, how about using `_` instead of `res` as the response of `GetBlockByHash`?